Lib/test/test_zipfile/_path/test_path.py

```diff
@@ -10,11 +10,11 @@
 import zipfile._path
 
 from test.support.os_helper import temp_dir, FakePath
+from test.support.parameterize import parameterize, Invoked
 
 from ._functools import compose
 from ._itertools import Counter
 
-from ._test_params import parameterize, Invoked
```

Comment:
Cosmetic detail: one more line should be removed here (to leave vertical space of 2 lines, rather than of 3 lines).